students/Hebdowska_Grazyna/lesson_08/email_validator.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+
+def email_validator(email):
+
+    s = r'''((\w{1,})(\.{1})*(\w{1,})*(\w[@]{1})(\w{1,})(\.{1})(\w{2,3}))'''
```

Comment:
{1,} is equivalent to +
{1} might be just omitted

It’s better to use *, +, or ? when you can, simply because they’re shorter and easier to read.